aio/karma.conf.js

```diff
@@ -66,7 +66,18 @@ module.exports = function(config) {
     colors: true,
     autoWatch: true,
     port: 9876,
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
I am not sure if we want to change the default to headless. Headless is used by the CI and for local development it's easier to debug with standard chrome.

Reply:
I added new config for container in next commit. Thank you for your reviewing. :)